src/webhook.ts

```diff
@@ -11,29 +11,61 @@ export const webhookEventTypeSchema = z.nativeEnum(WebhookEventType, {
   description: 'webhookEventTypeSchema',
 })
 
+const webhookEventBodyBaseSchema = z.object({
+  provider: z.string(),
+  eventId: z.string(),
+  timestamp: z.string(),
```

Comment:
Can we make this a regex and require exactly 10 digits?

Reply:
I think that is a great idea. Do you think it belongs in this pr? also do you think a change like that is a fix, feat, or breaking change?

Reply:
created this ticket for it https://linear.app/valora/issue/ACT-520/zod-schema-improvements